README.md

```diff
@@ -187,6 +186,22 @@ Flagship.define :production do
 end
 ```
 
+And you can also extend helper methods from base flagset.
+
+```rb
+Flagship.define :base do
+  def staff?(context)
+    def is_author(context)
```

Comment:
Ah sorry, it's just a mistake cased by my rewriting!
Fixed it.
https://github.com/yuya-takeyama/flagship/pull/22/commits/255500df04d4241603d6c0136fa4a9a9985c08dd